### PR TITLE
Remove deprecated QDesktopWidget

### PR DESCRIPTION
**Pull request by luis-pereira**
Maintaining functionality.

### CODE REVIEW

File: src/mainwindow.cpp [hunk 1]
```diff
@@ -34,6 +34,7 @@
 #include <QWheelEvent>
 #include <QMouseEvent>
 #include <QTimer>
+#include <QScreen>
```
Comment (tsujan) [inline src/mainwindow.cpp]: Would you please also remove `#include <QDesktopWidget>` (a few lines below)?

Comment (luis-pereira) [reply src/mainwindow.cpp]: sure. completely forgot it.

**Comment by luis-pereira** [comment]
@tsujan Done. Thanks.

**Comment by tsujan** [comment]
Thanks. GTM.

**Pull request merged**